pitch.md

```diff
@@ -23,38 +89,174 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+
+This is similar to how browsers will interpret CSS, some conflicts are allowed
+and the more specific directive is followed. I would anticipate trying to
+follow this behavior (though probably logging these incidents to the user could
+ensure they were intended).
+
+There is also the opportunity for conflicts with the same specificity, in which
+the best choice seems to be simply returning an error and reporting it to the
+user.
+
+There is also the semantic issue which could result if the combination of schedules
+finds no valid schedule. I'm not sure I would classify this as an error exactly,
+but certainly an unfortunate outcome which the code should be prepared to handle.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+I haven't made any changes to expressiveness since the last time I
+considered this idea.
+
+It should be easy to describe when are the great, good, bad, and terrible
+times to meet (likely default values of great and terrible respectively).
+It should also be possible to express more complicated dependencies,
+such as a preference for a meeting time that coincides or avoids another
+group member, or a freedom in a 3 hour block where only 1 block can be
+scheduled at a time.
+
+It should be very difficult to do perform functions unrelated to scheduling,
+such as arithmetic, nested control flow, or see implementation details.
+
+I don't think there should be anything in between (of medium difficulty).
+
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+As I mentioned there are definitely already DSLs in this domain:
+ * [WhenIsGood](http://whenisgood.net/)
+ * [When2Meet](http://www.when2meet.com)
+
+These are indeed DSLs because they are computerized schedulers
+with a specific syntax (painting over certain blocks), and associated
+semantics (painted blocks are free blocks).
+
+These DSLs are quite minimalistic (i.e. when they run, they only support the
+intersection operation for the sets of times from each individual -- as far
+as I can tell).  I am also looking at some of the features provided for
+[Premium Users](http://whenisgood.net/PremiumInfo).
+
+My solution to the scheduling problem might be a bit more
+ambitious in the functionality it hopes to provide.  This does allow for more
+expressiveness, which is nice, but expressiveness is a double edged sword,
+I will need to take care during design to avoid feature creep and extra complexity
+for users.  Perhaps I will find that the existing websites' simplicity is
+more desirable than the benefits of the expressiveness I aim to provide (but
+I hope not!).
```

Comment:
The lack of expressiveness in the existing solutions makes scheduling a much easier problem to solve from a technical standpoint, but not a practical one. 

As a side: in your solution, if a problem is unsatisfiable, you might present to the user the number of people free at each time similarly to how Doodle does so the user could ultimately pick.